src/content/docs/fr/reference/modules/astro-actions.mdx

```diff
@@ -132,6 +133,13 @@ L'utilitaire `isInputError()` permet de vérifier si une `ActionError` est une e
 
 <ReadMore>Consultez le [guide des erreurs de saisie de formulaire](/fr/guides/actions/#affichage-des-erreurs-de-saisie-du-formulaire) pour en savoir plus sur l'utilisation de `isInputError()`.</ReadMore>
 
+### `isActionError()`
+<p>
+**Type :** <code>(error?: unknown | <a href="#actionerror">ActionError</a>) => boolean</code><br/>
+<Since v="4.15.0" />
+</p>
```

Comment:
I'm nitpicking, but it's not spaced out enough 😅 This is mainly to keep the line numbers similar to easily find the changes to be made...
```suggestion
### `isActionError()`

<p>

**Type :** <code>(error?: unknown | <a href="#actionerror">ActionError</a>) => boolean</code><br/>
<Since v="4.15.0" />
</p>
```